reactive/kubernetes_master.py

```diff
@@ -1011,6 +1011,24 @@ def start_master():
     '''Run the Kubernetes master components.'''
     hookenv.status_set('maintenance',
                        'Configuring the Kubernetes master services.')
+
+    if not is_state('kubernetes-master.vault-kv.pending'):
```

Comment:
We're doing this to ensure we don't stomp on an encryption config file set up by the vault relation, right?  If so, I think it would be better to guard with `not is_state('kubernetes-master.secure-storage.created')`, or perhaps both flags -- not `vault-kv.pending` and not `secure-storage.created`.

I'm nervous that we'll enter `start_master` when vault has been configured, so it's no longer pending.  At that point, we'd overwrite whatever vault config we had with this.